Detectors/ITSMFT/ITS/base/include/ITSBase/GeometryTGeo.h

```diff
@@ -16,13 +16,20 @@ class TGeoPNEntry; // lines 17-17
 
 namespace AliceO2
 {
+  namespace ITSMFT
+  {
+    class Segmentation;
+  }
+}
+using AliceO2::ITSMFT::Segmentation;
```

Comment:
`using` statements in headers are forbidden by our policy (and for a good reason, IMHO).